pkg/tfvars/gcp/gcp.go

```diff
@@ -4,6 +4,7 @@ import (
 	"encoding/json"
 
 	gcpprovider "github.com/openshift/cluster-api-provider-gcp/pkg/apis/gcpprovider/v1beta1"
+	"github.com/openshift/installer/pkg/types"
```

Comment:
the imports are best organised like

```
<stdlib>

<external>

<installer>
```

Reply:
@abhinavdahiya PTAL and let me know if this is what you were saying.